po/zh_CN.po

```diff
@@ -214,8 +214,8 @@ msgid ""
 "via the 'status' command), none (assume every device can fence every machine)"
 msgstr ""
 "允许的值：Dynamic-List（通过“列表”命令查询设备），"
```

Comment:
Let's also keep the parameter value `dynamic-list` uncapitalized and the action name `list` here untranslated.